receiver/httpcheckreceiver/metadata.yaml

```
@@ -70,3 +70,38 @@ metrics:
       value_type: int
     unit: "s"
     attributes: [http.url, http.tls.issuer, http.tls.cn, http.tls.san]
+  httpcheck.dns_lookup.duration:
+    description: Time spent performing DNS lookup for the endpoint.
+    enabled: false
+    gauge:
+      value_type: int
+    unit: ms
+    attributes: [http.url]
+  httpcheck.tcp_connection.duration:
+    description: Time spent establishing TCP connection to the endpoint.
+    enabled: false
+    gauge:
+      value_type: int
+    unit: ms
+    attributes: [http.url]
+  httpcheck.tls_handshake.duration:
```

there's a section of the semantic conventions for TLS attributes no spans (https://github.com/open-telemetry/semantic-conventions/blob/main/model/tls/registry.yaml), i couldn't find details about handshake duration, but i would suspect it would deserve to have its own `tls.` namespace

```suggestion
  httpcheck.tls.handshake.duration:
```